prow/cmd/peribolos/OWNERS

```diff
@@ -4,5 +4,7 @@ reviewers:
 - cjwagner
 approvers:
 - cjwagner
+- MadhavJivrajani
+- Priyankasaggu11929
```

Comment:
Please also add me and @cblecker here :)

Reply:
updated. Thanks! :pray: